### PR TITLE
Issue 478: Fixed pysolc dependency for d3a python3 and pypy dockerfiles.

### PR DESCRIPTION
**Pull request by spyrostz**

### CODE REVIEW

File: Dockerfile_pypy [hunk 1]
```diff
@@ -9,5 +9,6 @@ RUN pip install -r/app/requirements/dev.txt && \
     pip install -e . && \
     pip install git+https://github.com/Jonasmpi/py-solc.git
 
-ENTRYPOINT ["pypy3", "src/d3a/d3a_core/launcher.py"]
+RUN which d3a
+ENTRYPOINT ["d3a"]
```
Comment (faizan2590) [inline Dockerfile_pypy]: Can you explain me these two lines?

Comment (spyrostz) [reply Dockerfile_pypy]: Thanks for noticing that, 'which d3a' is just a debugging command (in order to detect where d3a was installed). Therefore I will remove this. 
As for the entrypoint, the purpose is to use the CLI interface (via the 'd3a' command, and not the launcher script which is responsible for communicating via Redis)

**Comment by hannesdiedrich** [comment]
LGTM

File: Dockerfile_pypy (hunk 1 above)
Comment (faizan2590) [reply Dockerfile_pypy]: Thanks a lot for the explanation!

**Pull request merged**